src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -2191,11 +2348,17 @@
           short-summary: Number of days to retain logs.
         - name: --storage-account
           short-summary: Name or ID of the storage account in which to save the flow logs.
+    examples:
+        - name: Enable NSG flow logs
```

Comment:
Example text needs to end with `.`. May be more examples that I missed.

Reply:
Fixed in my branch